### ISSUE TITLE
Track Request for Vault Service DNS/SSL Certs

### ISSUE THREAD

**Issue by j-pye** [BCDevOps/OpenShift4-RollOut#280]

**Comment by manderson-it** [comment]
@j-pye the DNS names seem to be in place. I can resolve them.
Are the SSL certs done?

**Comment by j-pye** [comment]
Duplicate of #279?
Still waiting on SSL certs. 
iStore order is being processed. CSRs have been provided for both vault and vault-lab certificates.

**Comment by manderson-it** [comment]
All information for DNS and SSL certs is captured in the completed ticket [#279](https://github.com/BCDevOps/OpenShift4-RollOut/issues/279).
Closing this since it is a duplicate.
